libs/platform/src/lib/components/search-field/search-field.component.ts

```diff
@@ -457,6 +480,20 @@ export class SearchFieldComponent extends BaseComponent implements OnInit, OnDes
         this._dataSource = dataSource;
     }
 
+    /**
+     * @hidden return count for matching suggestion with input text
+     * @returns number
+     */
+    private _getSuggestionsLength(): number {
+        let count = 0;
+        this.suggestions?.forEach((suggestion) => {
+            if (this.inputText && suggestion.value?.toLowerCase().indexOf(this.inputText?.trim()?.toLowerCase()) > -1) {
+                count++;
+            }
+        });
+        return count;
```

Comment:
I'd recommend to place empty line before `return` for better readability

Reply:
fixed